shared/common-adapters/text.desktop.js

```diff
@@ -317,6 +318,11 @@ export const styles = {
     ...globalStyles.fontSemibold,
     ...sizeGroups['14']
   },
+  textBodySmallBold: {
```

Comment:
I think because the "•" is Lato Bold, it's a mistake on our end. I'm
guessing Zeplin takes the style of the 1st character. Let's do regular body
small!

On Wed, May 25, 2016 at 2:36 PM Marco Munizaga <notifications@github.com>
wrote:

> In shared/common-adapters/text.desktop.js
> <https://github.com/keybase/client/pull/2874#discussion_r64627975>:
>
> > @@ -317,6 +318,11 @@ export const styles = {
> >      ...globalStyles.fontSemibold,
> >      ...sizeGroups['14']
> >    },
> > +  textBodySmallBold: {
>
> Why does zeplin say Lato Bold?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2874/files/cca2fb8c3a5b6378396cb84a3ca45046ffe55bae#r64627975>
>